runatlantis.io/docs/custom-workflows.md

```diff
@@ -328,3 +328,4 @@ will execute a custom script containing the following code to create a SSH tunne
 `ssh -f -M -S /tmp/ssh_tunnel -L 3306:database:3306 -N bastion 1>/dev/null 2>&1`. Without
 the redirect, the script would block the Atlantis workflow.
 :::
+* If a workflow step returns a non-zero exit code, the workflow will stop. 
```

Comment:
Let's move this up so it's before the `:::` and so shows up in the same section as the rest of the notes.

Reply:
Sorry, my bad. Fixed now